misago/core/templatetags/misago_pagetitle.py

```diff
@@ -7,9 +7,9 @@
 @register.simple_tag
 def pagetitle(title, **kwargs):
     if "page" in kwargs and kwargs["page"] > 1:
-        title += " (%s)" % (_("page: %(page)s") % {"page": kwargs["page"]})
+        title += f" ({_('page: %(page)s') % {'page': kwargs['page']}})"
```

Comment:
Does this work with Django's `makemessages`?

Reply:
Not too familiar with django, apologies :/

Reply:
According to [Django documentation](https://docs.djangoproject.com/en/4.2/topics/i18n/translation/) this will break translations:

> https://docs.djangoproject.com/en/4.2/topics/i18n/translation/

Please refactor all lines where gettext functions are called within f-strings.